.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/buildroot/buildroot.git
 [submodule "wlutil/busybox"]
 	path = wlutil/busybox
-	url = https://git.busybox.net/busybox/
+  url = https://github.com/ucb-bar/busybox.git
```

Comment:
there's something funky with the indentation here

Reply:
fixed